python-checks/src/main/java/org/sonar/python/checks/tests/AssertOnTupleLiteralCheck.java

```diff
@@ -43,12 +43,18 @@ public void initialize(Context context) {
 
         var issue = ctx.addIssue(tuple, MESSAGE);
 
-        if (tuple.leftParenthesis() != null && tuple.rightParenthesis() != null) {
+        if (tuple.leftParenthesis() != null && tuple.rightParenthesis() != null && !tuple.elements().isEmpty()) {
```

Comment:
How do we raise on this test `assert () # Noncompliant` when the `tuple.elements` is not empty?

Reply:
I was thinking that it wouldn't make sense to offer a quick fix for `assert ()` as this would result in just `assert `. Or is that a sensible quick fix?

Reply:
Oh yes, this makes sense. I thought the check was at the level of the issue detection. And did not get why this would raise! Thanks for the clarification!